src/containers/ModelPanel/index.tsx

```diff
@@ -11,35 +14,32 @@ import {
     getUiDisplayDataTree,
     getIsNetworkedFile,
 } from "../../state/trajectory/selectors";
-import { getStatus } from "../../state/viewer/selectors";
-import { State } from "../../state/types";
 import {
-    getAgentVisibilityMap,
-    getAgentHighlightMap,
-    getRecentColors,
-    getSelectedAgentMetadata,
-} from "../../state/selection/selectors";
+    AgentRenderingCheckboxMap,
+    ChangeAgentsRenderingStateAction,
+    SetVisibleAction,
+    SetRecentColorsAction,
+    ApplyUserColorSelectionAction,
+} from "../../state/selection/types";
```

Comment:
are these all just rearrangements ?

Reply:
Yes, should be, the imports were just not in order and it was bothering me and making things hard to track.